core/camel-core/src/test/java/org/apache/camel/component/file/stress/FileAsyncStressManuallyManualTest.java

```diff
@@ -32,6 +31,10 @@
 @DisabledOnOs(OS.WINDOWS)
 public class FileAsyncStressManuallyManualTest extends ContextTestSupport {
```

Comment:
Same note about manual tests I wrote for `FileAsyncStressReadLockRenameManualManualTest`

Reply:
Thanks! 
I have reverted the changes.